### ISSUE TITLE
Pretrained Models

### ISSUE THREAD

**Issue by TheLostIn** [ggjy/DeFeat.pytorch#5]
Hi,thanks for your work. 
1. Can you share the pretrained models of Res101 and Res50?
2. And how is the setting of Res50(1/4)?

**Comment by ggjy** [comment]
-1- I use the pretrained models provided by MMDetection. For example, you can download them at ``https://open-mmlab.oss-cn-beijing.aliyuncs.com/pretrain/vgg16_caffe-292e1171.pth.``
-2- I train the Res50(1/4) with ``base_channels=16`` on ImageNet, the learning rate is similar with the official ResNet-50.

**Comment by jacktkk** [comment]
@ggjy 请问您可以共享Res50(1/4).pth这个模型参数吗？

**Comment by ggjy** [comment]
I have uploaded Res50(1/2) and Res50(1/4) to the [Baidu Cloud](https://pan.baidu.com/s/1TCWLfu-h5MM05-agiEpHrg), the access code is ujr3.
